src/main/resources/db/migration/hsqldb/V1_5__DROP_TABLE_feedback.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS dbo.feedback;
```

Comment:
If all variants (hsqldb and postgres) use the same migration, you can just use the common one.

Reply:
Resolved.